modules/contact-form/js/grunion-frontend.js

```diff
@@ -1,5 +1,6 @@
 jQuery( function ( $ ) {
-	if ( 'function' === typeof $.fn.datepicker ) {
-		$( '.contact-form input[type="date"]' ).datepicker( { dateFormat : 'yy-mm-dd' } );
+	var datefield = $( '.contact-form input[type="date"]' );
+	if ( 'function' === typeof $.fn.datepicker && 'text' === datefield[0].type ) {
+		datefield.datepicker( { dateFormat : 'yy-mm-dd' } );
```

Comment:
I'm not sure if hardcoding this date format is ideal -- we may be better off considering the blog's saved date format in wp options?  But as this is existing legacy behavior ... *shrug*

Reply:
Also, the html5 date type always formats as `yyyy-mm-dd` whereas we've been legacy using `yy-mm-dd` -- to avoid formatting mismatches we may want to add something on the form processing side that normalizes to two-digit years or change our internal format to match four digit years going forward.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date